manager/src/main/java/com/redhat/service/smartevents/manager/connectors/ConnectorsServiceImpl.java

```diff
@@ -51,7 +51,6 @@ public void createConnectorEntity(Processor processor, Action action) {
         persistConnectorEntity(processor, topicName, actionConnector.getConnectorType(), actionConnector.connectorPayload(action, topicName));
     }
 
-    @Transactional(Transactional.TxType.MANDATORY)
```

Comment:
@kostola Was there a reason why this is required?

Reply:
Replied [here](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/662#discussion_r859647623)